client/app/components/ApplicationArea/ApplicationLayout/VersionInfo.jsx

```diff
@@ -12,10 +13,10 @@ export default function VersionInfo() {
       {clientConfig.newVersionAvailable && currentUser.hasPermission("super_admin") && (
         <div className="m-t-10">
           {/* eslint-disable react/jsx-no-target-blank */}
-          <a href="https://version.redash.io/" className="update-available" target="_blank" rel="noopener">
+          <Link href="https://version.redash.io/" className="update-available" target="_blank" rel="noopener">
```

Comment:
Why use it for external links? (not saying we shouldn't - just trying to understand motiviation)

Reply:
First - to keep things consistent, so if we need a link next time - we just use the component and don't think if that is an external link or not.
Second - sometimes it was a bit hard to tell if URL in this particular place is local or external. To avoid issues I decided to use the component everywhere.
Third - customization. If one day we'll decide to, say, mark external links, or add some additional parameters to all local links, or whatever - we'll just need to update/override the Link component without searching all the places that should be updated.

Reply:
It makes it easier to add an event tracker too